Dockerfile

```diff
@@ -1,15 +1,23 @@
 FROM registry.opensuse.org/opensuse/leap:15.3
 
 ARG USER_ID=1000
-ARG GROUP_ID=1000
-ARG USER_NAME=foo
-
+ARG USER_NAME=ansible-obs
 
 RUN zypper -n install openssh-clients ansible ruby ruby-devel bash git libxml2-devel gcc make shadow vim curl
 RUN zypper -q clean -a
 
 RUN useradd -l -u ${USER_ID} -g users --home-dir /home/${USER_NAME} --create-home ${USER_NAME}
 
+RUN gem install bundler
```

Comment:
To ensure reproducible Docker images, we should pin the version of `bundler` we install.